LORIS_indepth.html

```diff
@@ -138,72 +67,21 @@ <h4 align="center"><font color="f27c30">Heterogenous data acquisition</font></h4
 			<h4 align="center"><font color="f27c30">Storage, processing & dissemination</font></h4>
 			</td>
 		</tr></table>
-
-	<img src="images/loris_versatility.png" align="center" width="90%" style="border:0">
-
-<!--	<table class = "reveal" style="padding:0;margin:0" width = "60%">
-		<tr>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/multisite_study.png" style="border:0">
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/longitudinal.png" style="border:0">
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/clinical_research.png" style="border:0">
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/public_dataset.png" style="border:0">
-			</td>
-			</tr></table> -->
-          </section>
-
-<!-- WHAT DOES LORIS DO? -->
-<section>
-            <h2>What does LORIS do?</h2>
-            <img src="images/LORIS_functionality.png">
-          </section>
-
-<!-- PERFERRED SLIDE WHAT DOES LORIS DO -->	  
-	  <section>
+            <img src="images/loris_versatility.png" align="center" width="90%" style="border:0">
+            <p><small>A flexible, customizable, and <font color="f27c30">open source</font> neuroimaging <br> database suitable for multiple types of projects.</small></p>
+            </section>          
+              </section>
+              
+              <!-- What does LORIS do? Column 2 -->
+              
+        <section>
+               <section>
 	    <h2>What does LORIS do?</h2>
 	    <img src="images/loris_purpose.jpg" style="border:0" width="80%"> 
```

Comment:
replace this slide with hexagon tiles from google slides  [link again here ] 
- create a second slide with  Loris map and circles (N subjects, X visits...)

Reply:
thanks, done